examples/knative/powercli/kn-pcli-tag/handler.ps1

```diff
@@ -73,7 +79,7 @@ Function Process-Handler {
 
    Write-Host "$(Get-Date) - Applying vSphere Tag `"$VCENTER_TAG_NAME`" to $vmName ...`n"
    try {
-      Get-VM $vmName | New-TagAssignment -Tag (Get-Tag -Name $VCENTER_TAG_NAME) -Confirm:$false
+      Get-VM $vmName | New-TagAssignment -Tag (Get-Tag -Name $VCENTER_TAG_NAME) -Confirm:$false -ErrorAction Stop | Out-Null
```

Comment:
Same comment as above, we should consider dropping Out-Null which will mask potentially useful output for debugging purposes